### ISSUE TITLE
Multiple Concierge on one system

### ISSUE THREAD

**Issue by Awarua-** [concierge/Concierge#275]
**In raising this issue, I confirm to the best of my ability the following (please check boxes, e.g. [X]):**

- [x] There are no duplicate issues.
- [x] The problem does not lie with an upstream dependency.
- [x] An installed module/integration/service/other is not causing your problem.
- [x] This is reproducible on the latest codebase with up to date NPM dependencies.
- [x] I have added appropriate labels to this issue.

**Expected Behaviour:**
I can run multiple instances of Concierge on the same system

**Actual Behaviour:**
One instance will run, the rest will crash on startup with a seg fault.

**Steps to Reproduce:**
Try to start more than one Concierge instance, from different directories

This is a problem for the future if we want horizontal scaling.
Work around atm, is to use docker containers.

**Comment by mrkno** [comment]
"I have added appropriate labels to this issue."
Ummm... bug? Regression?

**Comment by Awarua-** [comment]
Yes, it could be with upstream, but no real indication of what causes it.
It's also harder to track down now that the core is async.

**Comment by mrkno** [comment]
>  It's also harder to track down now that the core is async.

I will admit that. However it is async-await not raw promise so tracking down the source is possible

**Comment by mrkno** [comment]
![image](https://user-images.githubusercontent.com/6830312/30777943-5ffc963c-a123-11e7-9bb7-baed84e17fb8.png)

@Awarua- Aside from the obvious issue in the screenshot above (can't have 2 instances of boss running on same machine unless they bind to a different port), I can't reproduce on the following configuration:  
Node: v8.5.0
OS: Windows
Modules: default (2 fresh clones)
Config: default (2 fresh clones)
